plugins/modules/s3_bucket.py

```diff
@@ -1111,6 +1244,9 @@ def create_or_update_bucket(s3_client, module: AnsibleAWSModule):
         s3_client, module, name
     )
     result["object_lock_default_retention"] = bucket_object_lock_retention_result
+    # -- Inventory
+    bucket_inventory_changed, bucket_inventory_result = handle_bucket_inventory(s3_client, module, name)
+    result["bucket_inventory"] = bucket_inventory_result
```

Comment:
We should update the `RETURN` block to include this field, and maybe add an example to the `EXAMPLES` block for bucket inventory configuration.